3_javascript/javascript1/week1/javascript.js

```diff
@@ -0,0 +1,120 @@
+console.log('------------- Age - ify(A future age calculator)-------------');
+
+let yearOfBirth = 1976;
+let yearFuture = 2076;
+let age = yearFuture - yearOfBirth;
+
+console.log('You will be ' + age + ' years old in ' + yearFuture);
+console.log('\n');
+
+
+console.log('----------- Goodboy-Oldboy (A dog age calculator) -----------');
+let dogYearOfBirth = 2018;
+let dogYearFuture = 2036;
+let dogYear;
```

Comment:
I just love it when people give variables meaningful and understandable names. It makes it much easier to read and understand the code. That's so great, good job Katy 👍